pyproject.toml

```diff
@@ -1,71 +1,74 @@
-[tool.poetry]
-name = "entropylab"
-version = "0.1.2"
-description = ""
-license = "BSD-3-Clause"
-authors = [
-    "Guy Kerem <guy@quantum-machines.co>",
-    "Ilan Mitnikov <ilan@quantum-machines.co>",
-    "Gal Winer <gal@quantum-machines.co>",
-    "Tal Shani <tal@quantum-machines.co>"
-]
-packages = [
-    { include = "entropylab" }
-]
-include = [
-]
-exclude = [
-    "**/tests/**",
-]
-readme = "README.md"
-homepage = "https://github.com/entropy-lab/entropy"
-
-[tool.poetry.dependencies]
-python = "^3.7.1"
-sqlalchemy = "^1.4.0"
-panel = "^0.11.1"
-bokeh = "^2.3.0"
-param = "^1.10.1"
-dill = "^0.3.3"
-pandas = "^1.2.3"
-numpy = "^1.20.1"
-jsonpickle = "^2.0.0"
-graphviz = "^0.16"
-networkx = "^2.5.1"
-matplotlib = "^3.4.1"
-
-[tool.poetry.dev-dependencies]
-pytest = "^6.2.2"
-black = "^20.8b1"
-flake8 = "^3.9.0"
-flake8-bugbear = "^21.3.2"
-pytest-repeat = "^0.9.1"
-pytest-cov = "^2.11.1"
-poethepoet = "^0.10.0"
-
-[tool.pytest.ini_options]
-
-
-[tool.poe.tasks.format]
-cmd = "black entropylab"
-help = "Format source files according to the style rules"
-
-[tool.poe.tasks.check-format]
-cmd = "black entropylab --check"
-help = "Check that all files are formatted according to the style rules"
-
-[tool.poe.tasks.lint]
-cmd = "flake8 entropylab"
-help = "Check for lint errors"
-
-[tool.poe.tasks.test]
-cmd = "pytest"
-help = "Run all unit tests"
-
-[tool.black]
-target-version = ["py37"]
-line-length = 88
-
-[build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+[tool.poetry]
+name = "entropylab"

```

Comment:
Double check the line endings, I think they are windows lineendings,
just edit this file to be linux lineendings

Reply:
@Tal Done. Can I ask you to verify? 🙏

Reply:
Worked!